### PR TITLE
[WIP for 5.1] STAC-13408: Updated stackstate helm chart changes

### PR DESCRIPTION
**Pull request by zvheerden**

### CODE REVIEW

File: stackpacks/integrations/kubernetes.md [hunk 1]
```diff
@@ -70,40 +71,25 @@ For the Kubernetes integration to retrieve topology, events and metrics data, yo
...
-        path: /etc/stackstate-agent/conf.d/kubernetes_state.d
-        data: |
-          cluster_check: true
-  
-          init_config:
-  
-          instances:
-            - kube_state_url: http://YOUR_KUBE_STATE_METRICS_SERVICE_NAME:8080/metrics
+  dependencies:
+  kubeStateMetrics:
```
Comment (julianosk) [inline stackpacks/integrations/kubernetes.md:88]: Wrong indentation here

File: stackpacks/integrations/kubernetes.md [hunk 2]
```diff
@@ -116,20 +102,20 @@ To remedy this situation, the kubernetes\_state check can be configured to run a
...
   --set-string 'stackstate.url'='<STACKSTATE_RECEIVER_API_ADDRESS>' \
   --values values.yaml \
-  stackstate-cluster-agent stackstate/cluster-agent    
+  stackstate-agent stackstate/stackstate-agent
   ```
 
 ### Status
 
-To check the status of the Kubernetes integration, check that the StackState Cluster Agent \(`cluster-agent`\) pod and all of the StackState Agent \(`cluster-agent-agent`\) pods have status ready.
+To check the status of the Kubernetes integration, check that the StackState Cluster Agent \(`cluster-agent`\) pod and all of the StackState Agent \(`node-agent`\) pods have status ready.
```
Comment (julianosk) [inline stackpacks/integrations/kubernetes.md:110]: Add the Checks agent here.

**Review by julianosk — CHANGES_REQUESTED** [review]
It looks good, overall! I think there are a few things that could be improved.

File: setup/agent/kubernetes.md [hunk 3]
```diff
@@ -270,50 +267,109 @@ It is not necessary to configure this property if your cluster uses one of the d
...
-
-The following integrations have checks that can be configured to run as cluster checks:
-
-- **Kubernetes integration** - [Kubernetes_state check as a cluster check](/stackpacks/integrations/openshift.md#configure-cluster-check-kubernetes_state-check).
-- **OpenShift integration** - [OpenShift Kubernetes_state check as a cluster check](/stackpacks/integrations/openshift.md#configure-cluster-check-kubernetes_state-check).
-- **AWS integration** - [AWS check as a cluster check](/stackpacks/integrations/aws/aws.md#configure-the-aws-check).
-
 ### External integration configuration
 
 To integrate with other external services, a separate instance of the [StackState Agent](about-stackstate-agent.md) should be deployed on a standalone VM. Other than [cluster checks](#enable-cluster-checks), it is not currently possible to configure a StackState Agent deployed on a Kubernetes or OpenShift cluster with checks that integrate with other services.
```
Comment (gggina) [inline setup/agent/kubernetes.md:375]: links to section that has been removed

**Review by LouisParkin — APPROVED** [review]

**Pull request merged**